src/main/java/data/DataSet.java

```diff
@@ -50,4 +51,15 @@ public static Order addingOrder() {
             .setComplete(true);
     }
 
+    public static User addingUser() {
+        return new User()
+            .setId(311L)
```

Comment:
Is it necessary to set user id in request?

Reply:
@andyyarych no, not really. In FindUserByUsernameTest class I have test where I'm posting ID = null, and it's goes well.